### PR TITLE
Define IID_IAccessible in Interop IID

### PR DESCRIPTION
**Pull request by gpetrou**
## Proposed changes

- Define IID_IAccessible in Interop IID.

###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/dotnet/winforms/pull/6416)

### CODE REVIEW

**Comment by gpetrou** [comment]
Perhaps I should define Guid IID_IAccessible once instead in the Interop class?

**Comment by Tanya-Solyanik** [comment]
>Perhaps I should define Guid IID_IAccessible once instead in the Interop class?

Yes, great idea, please do!

File: src/System.Windows.Forms.Primitives/src/Interop/Oleacc/Interop.uuid_IAccessible.cs [hunk 1]
```diff
@@ -0,0 +1,13 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System.Runtime.InteropServices;
```
Comment (Tanya-Solyanik) [inline src/System.Windows.Forms.Primitives/src/Interop/Oleacc/Interop.uuid_IAccessible.cs:5]: Please move this line to fix the build

File: src/System.Windows.Forms.Primitives/src/Interop/Oleacc/Interop.uuid_IAccessible.cs [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+
+internal static partial class Interop
+{
+    internal static partial class Oleacc
+    {
+        public const string uuid_IAccessible = "{618736E0-3C3D-11CF-810C-00AA00389B71}";
```
[the remark this answers is not in the document]
Comment (RussKie) [reply src/System.Windows.Forms.Primitives/src/Interop/Oleacc/Interop.uuid_IAccessible.cs:11]: A bonus point for sorting these alphabetically.

File: src/System.Windows.Forms.Primitives/src/Interop/Oleacc/Interop.uuid_IAccessible.cs (hunk 1 above)
Comment (gpetrou) [reply src/System.Windows.Forms.Primitives/src/Interop/Oleacc/Interop.uuid_IAccessible.cs]: I have removed this file now.

**Review by Tanya-Solyanik — APPROVED** [review]
Thank you!

**Pull request merged**